### PR TITLE
Add NFTify to Showcase

### PR DESCRIPTION
**Pull request by thanhtran812**
## Checklist
* [x]  I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
* [x]  I have read the [Showcase Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js)
* [x]  I have run `yarn build` after adding my changes **without getting any errors**.

## Showcase addition
* Title: NFTify
* Description: The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch and trade their NFT assets without friction, coding skills – at the lowest possible costs.
* Website: https://nftify.network/
* Tags:
  * `marketplace`
  * `minting`
  * `nftproject`

### CODE REVIEW

**Review by rphair — COMMENTED** [review]
@thanhtran812 please first follow the recommendation on the repository front page: build the Portal to check your submission & then each time you make a structural change.

I haven't verified the appropriateness of all the tags but in a future commit I believe the wrong one was removed (see below).

Instead of a full screenshot, I would recommend using a detail of this web site for your cover image.  Thumbnails will display, at the most, about 320 pixels of width so people won't be able to see much more than the background splash.  How about changing to a zoom in on something both eye-catching _and_ functional with text readable at this size?

I also don't know how many submissions are appropriate for this category & since @katomm did a category trim recently maybe he can give us some idea about that. 🧐

File: src/data/showcases.js [hunk 1]
```diff
@@ -1127,6 +1127,16 @@ export const Showcases = [
...
   },
+  {
+    title: "NFTify",
+    description:
+      "The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch and trade their NFT assets without friction, coding skills – at the lowest possible costs.",
+    preview: require("./showcase/nftify.png"),
+    website: "https://nftify.network/",
+    source: null,
+    tags: ["marketplace", "minting", "nftproject"],
+    developmentCompany: "Ekotek",
```
Comment (rphair) [inline src/data/showcases.js:1138]:
```suggestion
```
Here is the error you need to fix: from this arbitrary, non-existent key.  I'll wait for you to commit this & ensure that it builds.
```
[ERROR] Docusaurus server-side rendering could not render static page with path /showcase/.
[ERROR] Docusaurus server-side rendering could not render static page with path /tools/.

Error: Showcase site with title=NFTify contains errors:
Site contains unknown attribute names=[developmentCompany]
```

File: src/data/showcases.js [hunk 2]
```diff
@@ -1127,6 +1127,16 @@ export const Showcases = [
     source: null,
     tags: ["lending", "token", "marketplace", "nftsupport"],
   },
+  {
+    title: "NFTify",
+    description:
+      "The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch and trade their NFT assets without friction, coding skills – at the lowest possible costs.",
```
Comment (rphair) [inline src/data/showcases.js:1133]: We don't have such promotional language for projects on the Dev Portal so please rewrite this as a straightforward list of features... especially those features that may be _unique_.

**Review by rphair — COMMENTED** [review]
p.s. my crappy Internet connection dropped my 2 comments so here they are again out of context in the above review:

File: src/data/showcases.js (hunk 1 above)
Comment (thanhtran812) [reply src/data/showcases.js]: @rphair Sorry my boss insist on having the company name attached with this :sweat_smile: . I have removed it, please check. Thank you!

File: src/data/showcases.js (hunk 2 above)
Comment (thanhtran812) [reply src/data/showcases.js]: Got it! I have changed the description.

**Comment by thanhtran812** [comment]
> @thanhtran812 please first follow the recommendation on the repository front page: build the Portal to check your submission & then each time you make a structural change.
> 
> I haven't verified the appropriateness of all the tags but in a future commit I believe the wrong one was removed (see below).
> 
> Instead of a full screenshot, I would recommend using a detail of this web site for your cover image. Thumbnails will display, at the most, about 320 pixels of width so people won't be able to see much more than the background splash. How about changing to a zoom in on something both eye-catching _and_ functional with text readable at this size?
> 
> I also don't know how many submissions are appropriate for this category & since @katomm did a category trim recently maybe he can give us some idea about that. 🧐

@rphair I have just changed the thumbnail into a 320px image and make some updated regarding your comments. Please check and let me know if there's any thing inappropriate. Thank you!

File: src/data/showcases.js (hunk 2 above)
Comment (rphair) [reply src/data/showcases.js]: I see it's changed, but not enough in my opinion.  Please tell your boss (re: https://github.com/cardano-foundation/developer-portal/pull/1204#discussion_r1477736962) by reference to this PR thread that we simply cannot list anything which is described with promotional language; currently:
> The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch an NFT store and trade their NFT assets",

Realistically: if you were the "leading" NFT marketplace, you would have been on the Developer Portal already.  I am speaking as another marketer who also recognises the need to keep things *simple and dry* when necessary.  As we generally insist & already said above, just stick to the features please.

Comment (thanhtran812) [reply src/data/showcases.js]: @rphair I have updated the description. Please check.

**Comment by rphair** [comment]
@thanhtran812 I appreciate your cooperation in standardising your listing.  I feel we wouldn't need another NFT creator but I think the number of white-label NFT creation services are fewer & therefore your service could be elegible.

It doesn't seem from the reading I've done that this service is connected to Cardano very much... I would invite other reviewers @rdlrt @katomm to see if they can verify that, and please also @thanhtran812 help me understand the Cardano connection.  Cardano isn't mentioned on the web site itself; only in a Document section: https://support.nftify.network/hc/en-us/sections/10339864961049-Cardano-network

... and the "white paper" (written & formatted like a sales pitch) also doesn't mention Cardano and says its Ethereum-based token is stakeable on a small number of chains which _don't_ include Cardano.

So I can't approve this listing unless another reviewer can take a second look and see a primary Cardano connection that I may have missed.  My understanding at this point is that this is an Ethereum based service that has been made compatible with Cardano, which doesn't itself qualify this service for inclusion in the Developer Portal.

**Review by rphair — COMMENTED** [review]
@thanhtran812 I appreciate your cooperation in standardising your listing.  I feel we wouldn't need another NFT creator but I think the number of white-label NFT creation services are fewer & therefore your service could be eligible.

It doesn't seem from the reading I've done that this service is connected to Cardano very much... I would invite other reviewers @rdlrt @katomm to see if they can verify that, and please also @thanhtran812 help me understand the Cardano connection.  Cardano isn't mentioned on the web site itself; only in a Document section: https://support.nftify.network/hc/en-us/sections/10339864961049-Cardano-network

... and the "white paper" (written & formatted like a sales pitch) also doesn't mention Cardano and says its Ethereum-based token is stakeable on a small number of chains which _don't_ include Cardano.

So I can't approve this listing unless another reviewer can take a second look and see a primary Cardano connection that I may have missed.  My understanding at this point is that this is an Ethereum based service that has been made compatible with Cardano, which doesn't itself qualify this service for inclusion in the Developer Portal.

**Comment by thanhtran812** [comment]
> @thanhtran812 I appreciate your cooperation in standardising your listing. I feel we wouldn't need another NFT creator but I think the number of white-label NFT creation services are fewer & therefore your service could be eligible.
> 
> It doesn't seem from the reading I've done that this service is connected to Cardano very much... I would invite other reviewers @rdlrt @katomm to see if they can verify that, and please also @thanhtran812 help me understand the Cardano connection. Cardano isn't mentioned on the web site itself; only in a Document section: https://support.nftify.network/hc/en-us/sections/10339864961049-Cardano-network
> 
> ... and the "white paper" (written & formatted like a sales pitch) also doesn't mention Cardano and says its Ethereum-based token is stakeable on a small number of chains which _don't_ include Cardano.
> 
> So I can't approve this listing unless another reviewer can take a second look and see a primary Cardano connection that I may have missed. My understanding at this point is that this is an Ethereum based service that has been made compatible with Cardano, which doesn't itself qualify this service for inclusion in the Developer Portal.

Hello @rphair,

I would like to address your assumption that Cardano isn't featured in NFTify's marketing materials. We can provide you with website links, introduction videos as well as access to our portfolio. You can take a look to see Cardano is indeed highlighted as one of the main networks alongside Polygon, Binance, and Ethereum.

* https://nftify.network/features/build-nft-store
* https://nftify.network/services/concierge
* https://www.youtube.com/watch?v=ZttnAm1h1T8
* https://www.youtube.com/watch?v=11SuV9dnykg&t=5s (2:58)
* Portfolio: https://docsend.com/view/txr95hhwqrkr8p9r (slide 13)

Btw, thank you so much for pointing out the absence of Cardano in our white paper. Actually, we no longer use this document to present to our customers or partners. We'll work on updating the white paper and publishing it on our website. I appreciate your time in reviewing our submission and look forward to hearing from you

**Comment by rphair** [comment]
@thanhtran812 yes, those additional materials might be helpful for other reviewers to establish enough of a relationship to Cardano for inclusion on the Developer Portal.  I still don't see a strong connection but I'll go along with other reviewers if they think the Cardano relationship is strong enough & will invite them to review your submission now.

**Comment by thanhtran812** [comment]
Dear @rphair, @rdlrt, @katomm, @os11k, and @fill-the-fill, 

I hope you guys had a chance to go through our submission and NFTify platform as well. we eagerly look forward to hearing your opinion and in case you need additional information from us to make the final decision, please don't hesitate to let me know.

**Pull request closed without merging**